src/components/photo/PhotoOrganizer.js

```diff
@@ -0,0 +1,55 @@
+import { useRef } from 'react'
+import { useDropzone } from 'react-dropzone'
+import styled from 'styled-components/macro'
+
+import { PhotoList } from './PhotoList'
+
+const StyledDropzone = styled.div`
+  border: 2px dashed ${({ theme }) => theme.orange};
+  border-radius: 12px;
+
+  font-size: 1.125rem;
+  cursor: pointer;
+
+  text-align: center;
+  vertical-align: middle;
+  height: 100px;
+  line-height: 100px;
+`
+
+export const PhotoOrganizer = ({ photos, onChange }) => {
+  const pendingPhotoId = useRef(0)
+
+  const onDrop = (acceptedFiles) => {
+    const newPhotos = acceptedFiles.map((file) => {
+      pendingPhotoId.current--
+
+      return {
+        id: pendingPhotoId.current,
+        name: file.path,
+        image: URL.createObjectURL(file),
+        isNew: true,
+        isUploading: true,
+        file,
+      }
+    })
+
+    onChange([...photos, ...newPhotos])
+  }
+
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept: 'image/*',
```

Comment:
It might be helpful to specify further, e.g. its possible to upload `.ico` files right now and be stuck on uploading even after the request fails.